File: CS/DXSample/Common/DXImageHelper.cs
using DevExpress.Xpf.Core;
using DevExpress.Xpf.Core.Native;
using System.Windows.Media;

namespace DXSample.Common {
    public static class DXImageHelper {
        public static ImageSource GetDXImage(string dxImageName) {
            DXImageExtension extension = new DXImageExtension();
            extension.Image = new DXImageConverter().ConvertFromString(dxImageName) as DXImageInfo;
            return extension.ProvideValue(null) as ImageSource;
        }
    }
}

File: VB/DXSample/Common/DXImageHelper.vb
Imports DevExpress.Xpf.Core
Imports DevExpress.Xpf.Core.Native
Imports System.Windows.Media

Namespace DXSample.Common

    Public Module DXImageHelper

        Public Function GetDXImage(ByVal dxImageName As String) As ImageSource
            Dim extension As DXImageExtension = New DXImageExtension()
            extension.Image = TryCast(New DXImageConverter().ConvertFromString(dxImageName), DXImageInfo)
            Return TryCast(extension.ProvideValue(Nothing), ImageSource)
        End Function
    End Module
End Namespace
